plugins/kafka/kafka.cpp

```diff
@@ -54,24 +58,80 @@ namespace KafkaPlugin
     // Static Methods (internal)
     //--------------------------------------------------------------------------
 
+#ifdef _CONTAINERIZED
```

Comment:
we normally try to compile both code alternatives, by using isContainerized(), unless we can't because it's dependent on bits of the system that are [e.g.] not in containerized.
With some hope that eventually almost all the code base will follow the 'containerized' approach.

In this case.. I guess it's too late for the code to be common?
e.g. could the BM system use yaml config files, and thus this applyConfigProps be common? Prob. not.

Reply:
Unfortunately, I don't think that would work. We do have active users on the bare metal side of things, so that configuration layout and usage of local files needs to stay in place.